src/docGeneratorMain.ts

```diff
@@ -35,7 +26,8 @@ async function main() {
   await docGenerator.downloadDocs(
     (pattern) =>
       `${githubBaseUrl}/EmmanuelDemey/eslint-plugin-angular/master/docs/rules/${pattern}.md`,
-    "angular"
+    "angular",
+    false
```

Comment:
It seems that we were mislead, due to the fact the old repo had some patterns. I updated the repo, and removed the false